Source/dafny.msbuild/build/dafny.msbuild.props

```diff
@@ -29,6 +29,7 @@
 
     <ItemGroup>
         <VerifyDafnyPassthrough Include="timeLimit:30" />
+		<CompileDafnyPassthrough Include="/coverage:.\bin\Debug\netcoreapp3.0\legend.txt" />
```

Comment:
This is perhaps a bit fragile, but that just means potential breakage if the build approach changes in the future, which I'm okay with.

Also, no tabs! :)